ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2561,5 +2561,75 @@
       "name": "type-apache-camel.svg",
       "mediaType": "image/svg+xml"
     }
+  },
+  {
+    "id": "python",
+    "name": "Python",
+    "creator": "Anatolii Bazko",
+    "tags": [
+      "Theia",
+      "Debian",
+      "Python",
+      "pip"
+    ],
+    "scope": "general",
+    "description": "Default stack with Python 3.7",
+    "components": [
+      {
+        "version": "9",
+        "name": "Debian"
+      },
+      {
+        "version": "3.7.3",
+        "name": "Python"
+      },
+      {
+        "version": "19.0.3 ",
+        "name": "PIP"
+      }
+    ],
+    "workspaceConfig": {
+      "environments": {
+        "default": {
+          "recipe": {
+            "content": "python:3.7.3",
+            "type": "dockerimage"
+          },
+          "machines": {
+            "python": {
+              "env": {
+                "PS1": "$(echo ${0})\\$ ",
+                "HOME": "/home/user"
+              },
+              "servers": {},
+              "volumes": {
+                "projects": {
+                  "path": "/projects"
+                }
+              },
+              "installers": [],
+              "attributes": {
+                "memoryLimitBytes": "512000000",
+                "containerCommand": "['sleep']",
+                "containerArgs": "['infinity']"
+              }
+            }
+          }
+        }
+      },
+      "commands": [],
+      "projects": [],
+      "defaultEnv": "default",
+      "name": "che7",
+      "attributes": {
+        "editor": "org.eclipse.che.editor.theia:next",
+        "plugins": "che-machine-exec-plugin:0.0.1,ms-python.python:2019.2.5433"
```

Comment:
I would explicitly change memory settings here: 512MB for Theia and 512MB for the language server:
```json
"ms-python.python.memory_limit": "512Mi",
"sidecar.org.eclipse.che.editor.theia.memory_limit": "512Mi"
```

Reply:
Do you mean
`"sidecar.ms-python.python.memory_limit": "512Mi",` ?